parquet-column/src/main/java/org/apache/parquet/io/RecordConsumerLoggingWrapper.java

```diff
@@ -112,7 +112,7 @@ public void addBoolean(boolean value) {
      */
     @Override
     public void addBinary(Binary value) {
-      if (DEBUG) log(Arrays.toString(value.getBytes()));
+      if (DEBUG) log(Arrays.toString(value.getBytesUnsafe()));
```

Comment:
getBytesUnsafe should always pay attention to the length + offset right? Should this be copy()?